### DISCUSSION TITLE
mbedTLS integration to Contiki

### DISCUSSION THREAD

**Discussion by chanjon** [contiki-ng/contiki-ng#1564] [Q&A]
I wonder anyone has attempted to integration mbedTLS into contiki and generate device CSR.

**Comment by nbrunner** [comment]
Try looking at this:
[https://www.diva-portal.org/smash/record.jsf?pid=diva2%3A1587410&dswid=6819](https://www.diva-portal.org/smash/record.jsf?pid=diva2%3A1587410&dswid=6819)
